eng/devices/ios.cake

```diff
@@ -91,6 +107,7 @@ Task("uitest")
 	});
 
 Task("cg-uitest")
+	.IsDependentOn("dotnet-buildtasks")
```

Comment:
control gallery?

Reply:
Ideally cg will be gone before the end of the month 😄